backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchBackendImpl.java

```diff
@@ -86,11 +85,10 @@
 		this.orchestratorFactory = new ElasticsearchWorkOrchestratorFactory( new LogErrorHandler(), client, gsonProvider, workFactory );
 		this.analysisDefinitionRegistry = analysisDefinitionRegistry;
 		this.multiTenancyStrategy = multiTenancyStrategy;
-		this.streamOrchestrator = orchestratorFactory.createStreamOrchestrator( name );
 		this.queryOrchestrator = orchestratorFactory.createStreamOrchestrator( name );
```

Comment:
You should pass a more meaningful name for the orchestrator. Something like `"Elasticsearch query orchestrator for backend " + name`.